prime-router/metadata/hl7_mapping/ORU_R01/base/xon-organization.yml

```diff
@@ -1,35 +1,45 @@
 name: ORU-R01-xcn-contact
 elements:
-  - name: organization-name
+  - name: xon-organization-name
     value: [ '%resource.name' ]
     hl7Spec: [ '%{hl7OrgField}-1' ]
 
-  - name: organization-type
+  - name: xon-organization-type
     resource: '%resource.extension(%`rsext-organization-name-type`)'
     condition: '%resource.exists()'
     value: [ '%resource.value.coding[0].code' ]
     hl7Spec: [ '%{hl7OrgField}-2' ]
 
-  - name: organization-type
+  - name: xon-organization-type
     resource: '%resource.extension(%`rsext-assigning-authority-namespace-id`)'
     condition: '%resource.exists()'
     value: [ '%resource.value' ]
     hl7Spec: [ '%{hl7OrgField}-6-1' ]
 
-  - name: organization-type
+  - name: xon-organization-type
     value: [ '%resource.extension(%`rsext-assigning-authority-universal-id`).value.getId()' ]
     hl7Spec: [ '%{hl7OrgField}-6-2' ]
 
-  - name: software-vendor-assigning-authority-type
+  - name: xon-software-vendor-assigning-authority-system
     value: [ '%resource.extension(%`rsext-assigning-authority-universal-id`).value.getIdType()' ]
     hl7Spec: [ '%{hl7OrgField}-6-3' ]
 
-  - name: organization-id-type
+  - name: xon-organization-id-type
     resource: '%resource.type.where(coding.system = ''http://terminology.hl7.org/CodeSystem/v2-0203'')'
     condition: '%resource.exists()'
     value: [ '%resource.coding[0].code' ]
     hl7Spec: [ '%{hl7OrgField}-7' ]
 
-  - name: organization-id
+  - name: xon-assigning-facility
+    condition: false # We have no data for this field yet
+    value: [ '' ]
+    hl7Spec: [ '%{hl7OrgField}-8' ]
+
+  - name: xon-name-representation-code
+    condition: false # We have no data for this field yet
+    value: [ '' ]
```

Comment:
Here as well

Reply:
Fixed the code to allow for empty FHIR paths, so it does not crash